x-pack/plugins/security_solution/server/config.ts

```diff
@@ -64,6 +64,12 @@ export const configSchema = schema.object({
    * Artifacts Configuration
    */
   packagerTaskInterval: schema.string({ defaultValue: '60s' }),
+
+  /**
+   * Detection prebuilt rules
+   */
+  prebuiltRulesFromFileSystem: schema.boolean({ defaultValue: true }),
+  prebuiltRulesFromSavedObjects: schema.boolean({ defaultValue: true }),
```

Comment:
I would add these switches to this file:
https://github.com/elastic/kibana/blob/master/src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

even though we aren't planning on exposing this publicly or for cloud.

Reply:
fixed in 8f9eb8247e42362f5b6328eb1c567100c179a864